charbot/events.py

```diff
@@ -407,7 +407,7 @@ async def on_message(self, message: discord.Message) -> None:
         if not await self.sensitive_scan(message):
             return
         author = cast(discord.Member, message.author)
-        if all(
+          if all(
```

Comment:
```suggestion
        if all(
```
Why was this indented?